pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -28,6 +28,8 @@ const (
 	// This should remain in the default role bindings for the NodeAdmin role
 	LegacyMasterKubeletAdminClientUsername = "system:master"
 	MasterKubeletAdminClientUsername       = "system:openshift-node-admin"
+
+	InfraNodeBootstrapServiceAccountName = "node-bootstrapper"
```

Comment:
@enj indeed the controller_policy file now risk becoming a kitchen sink, feels wrong to me.
That is why I put the constants in ... guess what? "constants.go"